example/tutorials/code/rtt_iris.py

```diff
@@ -0,0 +1,107 @@
+import os
+import math
+import matplotlib.pyplot as plt
+import tensorflow as tf
+import numpy as np
+import pandas as pd
+
+import latticex.rosetta as rtt
+
+np.set_printoptions(suppress=True)
+
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
+
+np.random.seed(0)
+
+EPOCHES = 1
+BATCH_SIZE = 32
+learning_rate = 0.003
+
+rtt.activate("SecureNN")
+mpc_player_id = rtt.py_protocol_handler.get_party_id()
+
+# real data
+# ######################################## difference from tensorflow
+training_features_file_path = "/home/juzix/RosettaPlayground/example/tutorials/dsets/P" + str(mpc_player_id) + "/iris_training_features.csv"
```

Comment:
DO NOT use absolute path of your local machine in source code.

Reply:
OK

Reply:
These codes under the `code` directory should be removed since there has been a copy in the directory `contrib`.